app/javascript/components/home/NavbarSignedIn.jsx

```diff
@@ -25,10 +25,20 @@ import PropTypes from 'prop-types';
 import { ChevronDownIcon } from '@heroicons/react/20/solid';
 import useDeleteSession from '../../hooks/mutations/sessions/useDeleteSession';
 import Avatar from '../users/user/Avatar';
+import useSiteSetting from '../../hooks/queries/site_settings/useSiteSetting';
 
 export default function NavbarSignedIn({ currentUser }) {
   const { t } = useTranslation();
   const deleteSession = useDeleteSession({ showToast: true });
+  const { data: helpCenter } = useSiteSetting('HelpCenter');
+
+  // check if Help Center link is set in Site Settings and return link
+  const getHelpCenterLink = () => {
+    if (helpCenter) {
+      return helpCenter;
+    }
+    return 'https://docs.bigbluebutton.org/greenlight/v3/install';
```

Comment:
I think its better if we remove this as the default personally

Reply:
Just hide it if not set

Reply:
Made the update to just hide the nav link if help center not set!